test.kt

```diff
@@ -12,8 +12,6 @@ fun myFunction() {
     println("line 12")
     println("line 13")
     println("line 14")
```

Comment:
This is a multi-line code suggestion
```suggestion
    println("line 13") // line 13
    println("line 14")
```